app/javascript/packages/address-search/components/address-input.spec.tsx

```diff
@@ -5,7 +5,7 @@ import { setupServer } from 'msw/node';
 import { rest } from 'msw';
 import type { SetupServer } from 'msw/node';
 import { SWRConfig } from 'swr';
-import AddressSearch from '.';
+import AddressSearch from './address-input';
```

Comment:
```suggestion
import AddressInput from './address-input';
```
This works because it lets you assign any name to the default export from `address-input`, but shouldn't it be `AddressInput`? (here and two places in the body of the test)